views/transaccion_completa_mall_deferred/create.pug

```diff
@@ -37,13 +37,13 @@ block step
         | Ahora que ya creamos la transacción tienes dos opciones, puedes hacer consultas de cuotas para pagar con ellas
         | o puedes directamente confirmar la transacción
   .row
-    form.col.m12#confirmForm(action="/transaccion_completa_mall_deferred/commit", method="POST")
```

Comment:
is correct change to _transaccion_completa_mall_, if this is _transaccion_completa_mall_deferred_ view?